app/assets/javascripts/discourse/tests/acceptance/composer-test.js

```diff
@@ -102,6 +102,7 @@ acceptance("Composer", function (needs) {
     const event = document.createEvent("Event");
     event.initEvent("keydown", true, true);
     event[mac ? "metaKey" : "ctrlKey"] = true;
+    event.key = "B";
     event.keyCode = 66;
```

Comment:
Do we still need to set the `keyCode` here?

Reply:
This more closely imitates the real event, and some of our dependencies still rely on `keyCode`.
So in cases where they may act on those synthetic events we have to provide both properties.